apps/docs/docs/user/examples/README.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 10
+sidebar_position: 11
```

Comment:
I assume that was just wrong and you corrected it

Reply:
Depends, we had two entries on sidebar position 10, so I assume they were ordered by alphabet. This switches the order and moves examples down so that use cases and examples are next to each other.